bot.py

```diff
@@ -14,9 +15,11 @@
 
 try:
     import config
+    import utils#from modules import utils
```

Comment:
Don't think these needs to be in a try, and comment can be removed (utils isn't a cog anyway)?